include/deal.II/lac/full_matrix.h

```diff
@@ -163,11 +163,6 @@ class FullMatrix : public Table<2,number>
     const_iterator &operator++ ();
 
     /**
-     * Postfix increment.
-     */
-    const_iterator &operator++ (int);
```

Comment:
I think this may be a case where it's useful to provide an implementation, rather than remove the declaration. That's because it is often a personal question of style whether one writes `++it` or `it++` and currently one of these will lead to linker errors whereas after the change it will lead to a compiler error -- neither seems desirable.

Reply:
Fair point. I added this back in (and added a test).